src/middlewares/isSuperAdmin.js

```diff
@@ -0,0 +1,23 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+import RoleService from "../services/roleServices";
+
+
+export function isSuperAdmin (req, res, next){
+
+    const authHeader = req.headers['authorization']
```

Comment:
can we add this logic in auth middleware (I guess it there) and add the token in headers that were passing via next()